svc-aca-py/util/start_wallet_from_seed.sh

```diff
@@ -89,7 +84,6 @@ aca-py start \
 --webhook-url "${WEBHOOK_URL}" \
 --genesis-url "${LEDGER_URL}" \
 --label "${LABEL}" \
---public-invites \
```

Comment:
Here I'm not really sure what are reasons or benefits to use this flag.
As I understand the DID connection will still be created on a new local DID. This only affects the DID advertised in the initial connection invitation.
https://github.com/hyperledger/aries-cloudagent-python/issues/358#issuecomment-581503697

Using this flag might make sense in a more PoC / demo environment to help track and understand who is initiating the invite.
This flag is also necessary to accept unsolicited connection requests, not a use cases that should be expected by default:
https://github.com/hyperledger/aries-cloudagent-python/blob/a848bcbf2ec515f612afec7e2a01e82ab406cbe8/aries_cloudagent/config/argparse.py#L1073